frameworks/kafka/universe/config.json

```diff
@@ -306,7 +306,7 @@
           "default": true
         },
         "kafka_advertise_host_ip": {
-          "description": "Automatically configure advertised.host.name with the tasks' IP address",
+          "description": "DEPRECATED: This configuration parameter has been deprecated and will be removed in a future release. When set to true, and no security settings are enabled, kafka's advertised.listeners property is set to PLAINTEXT://<broker-ip>:<broker-port>. When the setting is false and no security is enabled, then advertised.listeners is NOT set. When security is enabled, the advertised.listeners property will be set to broker hostnames.",
```

Comment:
I think in a recent HDFS PR we used `[DEPRECATED]` as the marker for deprecation.

Reply:
I don't see any `[DEPRECATED]` markers in hdfs config.json?

Reply:
The fields intended for deprecation were removed from master and marked as deprecated on `sdk-0.40`: https://github.com/mesosphere/dcos-commons/blob/sdk-0.40/frameworks/hdfs/universe/config.json#L1358

Reply:
Ah, thanks!

I'm going to leave the deprecated fields in master as this field will probably stick around awhile